src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/NetTcpBindingElement.cs

```diff
@@ -126,6 +126,9 @@ public override Binding CreateBinding()
                 HostNameComparisonMode = HostNameComparisonMode
             };
 
+            //this.ReliableSession.ApplyConfiguration(nptBinding.ReliableSession);
+            Security.ApplyConfiguration(binding.Security);
+            //this.ReaderQuotas.ApplyConfiguration(nptBinding.ReaderQuotas);
             return binding;
```

Comment:
The ReaderQuotas are being applied on line 124 so you can remove this line. Although XmlDictionaryReaderQuotasElement.Clone() should really be called something like CreateReaderQuotas() as it's not really cloning. Up to you if you want to fix that naming, it's internal anyway.